### PR TITLE
Windows specific updates to docs

### PR DESCRIPTION
**Pull request by remarkov**
open for discussion re wording

### CODE REVIEW

File: content/troubleshooting/accessing-builder-machine-via-ssh.md [hunk 1]
```diff
@@ -64,8 +63,8 @@ If you have finished debugging the build, don't forget to end the session to avo
 
 * You can access your project on the builder machine by running `cd $FCI_BUILD_DIR`
 * If you encounter an error during the build, a good way to get started is by digging into the build logs on Codemagic and rerunning the first failed command.
-* The command `printenv` prints all the environment variables exported during the build. See [environment variables](../building/environment-variables/) for an explanation of read-only environment variables.
-* The `sudo` command is available to execute any commands with root privileges.
+* The command `printenv` prints all the environment variables exported during the build on Linux/macOS. Use `dir env:` on Windows to achieve the same in Powershell. See [environment variables](../building/environment-variables/) for an explanation of read-only environment variables.
+* The `sudo` command is available to execute any commands with root privileges on Linux/macOS. PowerShell on Windows machines is already running in privileged mode.
```
Comment (remarkov) [inline content/troubleshooting/accessing-builder-machine-via-ssh.md:67]: the first four tips are just repeating what is already said just a few paragraphs above, should we have twice?

File: content/troubleshooting/accessing-builder-machine-via-ssh.md [hunk 2]
```diff
@@ -64,8 +63,8 @@ If you have finished debugging the build, don't forget to end the session to avo
...
 * During an active SSH session, you can rerun the build with a different configuration right from the terminal.
```
Comment (remarkov) [inline content/troubleshooting/accessing-builder-machine-via-ssh.md:68]: not really clear what does this mean. it gives an impression that you may rerun the CM build from the terminal which is not the case I suppose.

**Review by mikhail-tokarev — APPROVED** [review]
LGTM

File: content/troubleshooting/accessing-builder-machine-via-ssh.md [hunk 3]
```diff
@@ -64,8 +63,8 @@ If you have finished debugging the build, don't forget to end the session to avo
 
 * You can access your project on the builder machine by running `cd $FCI_BUILD_DIR`
 * If you encounter an error during the build, a good way to get started is by digging into the build logs on Codemagic and rerunning the first failed command.
-* The command `printenv` prints all the environment variables exported during the build. See [environment variables](../building/environment-variables/) for an explanation of read-only environment variables.
-* The `sudo` command is available to execute any commands with root privileges.
+* The command `printenv` prints all the environment variables exported during the build on Linux/macOS. Use `dir env:` on Windows to achieve the same in Powershell. See [environment variables](../building/environment-variables/) for an explanation of read-only environment variables.
```
Comment (VeArnold) [inline content/troubleshooting/accessing-builder-machine-via-ssh.md:66]: a small nitpick, PowerShell*

Comment (remarkov) [reply content/troubleshooting/accessing-builder-machine-via-ssh.md]: thanks, fixed

**Pull request merged**